openeo/extra/spectral_indices/resources/spectral-indices-dict.json

```diff
@@ -0,0 +1,1344 @@
+{
+    "SpectralIndices": {
+        "ANIR": {
+            "bands":
+            [
+                "R",
+                "N",
+                "S1"
+             ],
+            "contributor": "vito",
+            "date_of_addition": "2021-10-27",
+            "formula": "exec('import numpy as np') or exec('from openeo.processes import clip') or np.arccos(clip((( np.sqrt( (0.8328 - 0.6646)**2 + (N - R)**2 )**2 + np.sqrt( (1.610 - 0.8328)**2 + (S1 - N)**2 )**2 - np.sqrt( (1.610 - 0.6646)**2 + (S1 - R)**2 )**2 ) / (2 * np.sqrt( (0.8328 - 0.6646)**2 + (N - R)**2 ) * np.sqrt( (1.610 - 0.8328)**2 + (S1 - N)**2 ))), -1,1)) * (1. / np.pi)",
+            "long_name": "Angle at Near InfraRed",
+            "reference": "",
+            "short_name": "ANIR",
+            "type": "vegetation",
+            "range": "(0,1)"
```

Comment:
is this `range` a customization? I can't find it in original https://github.com/davemlz/eemont/blob/master/eemont/data/spectral-indices-dict.json

Reply:
Yes, I need that for rescaling in compute_indices

Reply:
also:  avoid strint `"(0,1)"`, just specify an array with two elements